eng/scripts/InstallVisualStudio.ps1

```diff
@@ -28,13 +28,13 @@
     https://visualstudio.com
     https://github.com/dotnet/aspnetcore/blob/main/docs/BuildFromSource.md
 .EXAMPLE
-    To install VS 2022 Enterprise, run this command in PowerShell:
+    To install VS 2022 Community, run this command in PowerShell:
 
         .\InstallVisualStudio.ps1
 #>
 param(
     [ValidateSet('BuildTools','Community', 'Professional', 'Enterprise')]
-    [string]$Edition = 'Enterprise',
+    [string]$Edition = 'Community',
```

Comment:
@hishamco Just to confirm: you were able to successfully use the Community version of VS to develop on the repo?

Reply:
I will try to do something will localization, but seems it make no deference because all the projects have been loaded successfully